ingestion/src/test/java/feast/ingestion/TestFeatureRow.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
These test files should not be here. If you do need to have test files (although i think this is unnecessary since feature rows could easily be declared in code) they should be placed in the `test/resources` folder.